module/configs/pych.json

```diff
@@ -102,7 +102,7 @@
             ],
 
             "commands": [
-                "chpl %(cflags)s %(ctargets)s -o __TMP_PATH__",
+                "chpl %(cflags)s __DEPENDS__ %(ctargets)s -o __TMP_PATH__",
```

Comment:
Looking at this, it seems like the `-M <dir>` flags should be added to `cflags`. Is there something that prevents that?